src/test/ui/parser/else-no-if.stderr

```diff
@@ -0,0 +1,54 @@
+error: expected `{`, found keyword `false`
+  --> $DIR/else-no-if.rs:3:7
+   |
+LL |     } else false {
+   |       ^^^^
```

Comment:
```suggestion
   |            ^^^^^
```

Shouldn't the unexpected _expression_ (here: `false`) be highlighted to be in line with the diagnostic message? The `else` is not the unexpected token and thus shouldn't have a primary highlight (maybe only a secondary one for context).
Alternative (with secondary span):

```suggestion
   |       ---- because of this `else`
```